test/addons/openssl-client-cert-engine/test.js

```diff
@@ -46,9 +46,9 @@ const server = https.createServer(serverOptions, (req, res) => {
   const req = https.request(clientOptions, common.mustCall(function(response) {
     let body = '';
     response.setEncoding('utf8');
-    response.on('data', function(chunk) {
+    response.on('data', common.mustCall(function(chunk) {
```

Comment:
This is likely a bug. If the data comes across in multiple chunks, mustCall will complain that it has been called too often. Only the mustCall for the end event is necessary for data confirmation because at that point, the dats is compared to the expected result, so the arrival of the data even is implicitly confirmed (and the data check doesn't depend on how many data events occur, as long as all the chunks arrived).

Reply:
Thanks for pointing out, updated.